android/src/main/java/com/mapbox/mapboxgl/MapboxMapController.java

```diff
@@ -302,6 +303,7 @@ public void onMapReady(MapboxMap mapboxMap) {
     });
 
     setStyleString(styleStringInitial);
+    mapboxOfflineManager = new MapboxOfflineManager(context,this.mapboxMap,registrar);
```

Comment:
I don't know exactly the details, but is this map controller destroyed on rotation?
Would this mean that `MapboxOfflineManager` will also be destroyed with all it's callbacks inside loosing the download state?

Reply:
It's not destroyed, but it would be preferable to be able to download regions independent from any maps. For example, a dev might want to always download a certain city the first time a "local guide" app is launched.